tests/FakeItEasy.IntegrationTests/Assertions/ExceptionMessagesTests.cs

```diff
@@ -33,7 +33,7 @@ public void Exception_message_should_be_correctly_formatted()
     1: FakeItEasy.IntegrationTests.IFoo.Bar() 2 times
     ...
     3: FakeItEasy.IntegrationTests.IFoo.Bar(argument: ""test"")
-    4: FakeItEasy.IntegrationTests.IFoo.Bar(argument: 4/5/1977 12:00:00 AM, argument2: ""birthday"")
+    4: FakeItEasy.IntegrationTests.IFoo.Bar(argument: 4/5/*77 12:00:00 AM, argument2: ""birthday"")

```

Comment:
> keep everything like you have it, because it's fine

That's one of my favorite options :)

> just not test with a Date. I don't feel like it's an integral part of the test

Probably the best thing to do.

>     use culture `""` instead of `"en-US"`

I don't think it makes a difference... The format used for short date is based on the system locale, which uses two digits for the year

>     add an `ArgumentValueFormatter<Date>` to ensure a consistent format. Maybe `ToString("s", System.Globalization.CultureInfo.InvariantCulture)`

I don't think it would work, for the same reason as above. We'd  have to specify the full format explicitly.

>     do that, but in the core library. It's a change, but maybe even a feature! (We could slip that in a separate PR for visibility.)

I'm not convinced it would be a good thing... I think people like having dates in their own culture in error messages.

Reply:
> I think people like having dates in their own culture in error messages.

Heathens.